AppAuth-JS/package-lock.json

```diff
@@ -0,0 +1,5214 @@
+{
```

Comment:
This is generated source file. package-lock.json file is generated based on package.json to resolve dependencies. As standard, it is also committed